receiver_test.go

```diff
@@ -356,7 +356,7 @@ func TestReceiveSuccessModeFirst(t *testing.T) {
 		t.Fatalf("unexpected unsettled count %d", c)
 	}
 	// wait for the link to unpause as credit should now be available
-	assert.NoError(t, waitForLink(r.link, false))
+	require.NoError(t, waitForLink(r.link, false))
```

Comment:
I suspect there are more places where I wrote `assert` but meant `require` :(

Reply:
Luckily only annoying and not fatal :)